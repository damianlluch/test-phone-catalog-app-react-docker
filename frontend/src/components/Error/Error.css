.Error {
  display: flex;
  justify-content: center;
  text-align: center;
}

.Error h2 {
  color: red;
}
