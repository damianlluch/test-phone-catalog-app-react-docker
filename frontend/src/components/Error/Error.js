import React from 'react';
import PropTypes from 'prop-types';
import './Error.css';

const Error = props => (
  <div className="Error">
    <h2>{props.error}</h2>
  </div>
);

Error.propTypes = {
  error: PropTypes.string,
};

export default Error;
