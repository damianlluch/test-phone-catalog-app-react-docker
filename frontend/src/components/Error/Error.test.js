import React from 'react';
import renderer from 'react-test-renderer';
import Error from './Error';

describe('<Error/>', () => {
  it('renders by default', () => {
    const tree = renderer.create(<Error error="" />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
