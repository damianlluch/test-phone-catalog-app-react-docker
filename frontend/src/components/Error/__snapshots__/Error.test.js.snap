// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error/> renders by default 1`] = `
<div
  className="Error"
>
  <h2>
    
  </h2>
</div>
`;
